src/main/scala/kiteytys/email/Mailer.scala

```diff
@@ -4,64 +4,100 @@ import javax.mail.util.ByteArrayDataSource
 
 import com.typesafe.scalalogging.LazyLogging
 import kiteytys.conf.Conf
-import org.apache.commons.mail.{EmailException, EmailAttachment, DefaultAuthenticator, MultiPartEmail}
+import kiteytys.Game
+import org.apache.commons.mail.{EmailException, EmailAttachment, DefaultAuthenticator, MultiPartEmail, SimpleEmail, Email}
 
 import scalaz.concurrent.Task
 
 final class Mailer(conf: Conf.Smtp) extends LazyLogging {
 
-  val subject = "Kiteyttäjä"
-  val message = """Moi!
-    |
-    |Toivottavasti pelihetki oli antoisa - tässä Kiteyttäjä. Huomaat Kiteyttäjän visuaalisen kauneusleikkauksen etenemisen tässä matkan varrella.
-    |
-    |Me uskomme palautteen voimaan. Haasta meidät palautteella niin tulet huomaamaan kehityksen seuraavaan pelisessioon mennessä.
-    |
-    |Risut, ruusut ja pajut voi ohjata esimerkiksi johonkin seuraavista:
-    |
-    |galla@galliwashere.com
-    |0400246626
-    |@jussigalla
-    |
-    |innolla,
-    |gällit
-    |
-    |Play. Focus. Do. Repeat""".stripMargin
-
-  val attachmentName = "Kiteyttaja.pdf"
-  val attachmentDescription = "PDF"
-  val attachmentMime = "application/pdf"
-
   def sendPDF(recipient: String, data: Array[Byte]): Task[String] = {
-    val email = dataToEmail(recipient, data)
-    try {
-      val result = email.send()
-      logger.info(s"Sent email to ${recipient}. Message ID: $result")
-      Task.now(result)
-    } catch {
-      case ex: EmailException => Task.fail(ex)
-      case ex: IllegalStateException => Task.fail(ex)
-    }
+    val email = new MultiPartEmail()
+
+    val subject = "Kiteyttäjä"
+    val message = """Moi!
```

Comment:
Static values can be lifted to an object:

```scala
object Mailer {
  val message = """asdf"""
}

final class Mailer(conf: Conf.Smtp) extends LazyLogging {
...
```

On the other hand, I think this text should probably be loaded from a text file or something. Typesafe config probably supports multiline strings.